src/session.js

```diff
@@ -24,6 +25,11 @@ export type MessageResponse<T> = {
 
 export type DefaultMessageResponse = MessageResponse<Object>;
 
+/* eslint-disable no-redeclare */
+declare function coinNetwork(coin: string | bitcoin.Network): bitcoin.Network;
+declare function coinNetwork(coin: CoinInfo): CoinInfo;
```

Comment:
ugh this is rather complicated :D I think I simplified all this in the PRs that also use the new bitcoinjs-trezor... but since we don't have this merged, I guess it's OK?

Reply:
the problem is that we got all the three types here just because of backwards compatibility with some old version if mytrezor that doesn't exist anymore. We can very well get rid of all the different input types because we are the only ones using these libraries.... that's what I did in the waiting PRs

but anyway if this seems to be working now,,,

Reply:
yes, i described this in comment below and also in bitcoinjs-trezor issue. Coin definition needs to be changed in the whole stack (starts from bitcoinjs-trezor thru trezor.js and ends in mytrezor) to avoid unnecessary conversion from "xpub" to "ypub" back and forth.

Reply:
I was trying to do it as simple as i could, i'm aware it's just a workaround. But we don't want to "unscrew the whole tractor" right now :)